config/locales/en.yml

```diff
@@ -68,7 +68,6 @@ en:
       command_paper_number: Attachment command paper number
       hoc_paper_number: House of Commons paper number
       isbn: Attachment ISBN
-      price: Attachment price
```

Comment:
Here too. Is this something that's used in the frontend or whitehall-admin?

Reply:
I think that by removing this, we are losing the internationalisation on the frontend.

Reply:
I don't think so, I've removed all mention of order_url and I didn't take out any internationalisation, I think it's probably just in here to mirror the model.

Reply:
actually scratch that, I've just spotted something that I missed I think I need to change something around a bit. I think it's safe to remove `order_url` and `price` in the way that I have because they're only used when inputting but I need to use `order_a_copy_full` when presenting the link.

Reply:
Whilst we're here, can you move the I18n removals to separate commits? There's a lot of them, so you have to scroll a long way to see which tests have been removed.